packages/inspire/Revelation.js

```diff
@@ -1,6 +1,7 @@
 // @flow
 
 import { dumpObject, inProduction, isPromise, request, wrapError } from "~/tools";
+import isInBrowser from "is-in-browser";
```

Comment:
Let's extract the is-in-browser implementation to ~/tools/wrapError.js:inBrowser and use it everywhere